File: Adjust_Intensity_Values.m
i = imread('cameraman.jpg');      % Read the image
j = imadjust(i,[0 1],[1 0]);  % (image, [Low_in High_in], [Low_out High_out])

subplot(1,2,1);
imshow(i);
title('Original')

subplot(1,2,2);
imshow(j);
title('Adjusted')

File: Adjust_Intensity_Values.py
#!/usr/bin/env python
# coding: utf-8

# In[1]:


from skimage.io import imread
from skimage import exposure
import matplotlib.pyplot as plt


i = imread('cameraman.jpg')
j = exposure.rescale_intensity(i, in_range=(0, 125), out_range=(0, 1))

fig, axs = plt.subplots(1, 2, figsize=(10, 5))
axs[0].imshow(i)
axs[0].set_title('Original')

axs[1].imshow(j)
axs[1].set_title('Adjusted')
plt.axis('off')

plt.show()

